### PR TITLE
Added skeleton to manage tokens in asto

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #57 
Added `Tokens` interface and asto implementation, skeleton to manage tokens in asto.

### CODE REVIEW

**Comment by olenagerasimova** [comment]
@g4s8 please, take a look, that's just a skeleton, what do you say about design?

File: src/main/java/com/artipie/conda/Tokens.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+        /**
+         * Token.
+         */
+        private final String token;
+
+        /**
+         * Expiration date.
+         */
+        private final Date expire;
```
Comment (g4s8) [inline src/main/java/com/artipie/conda/Tokens.java:57]: I'd use `Instant` here, as a replacement for `Date`. Old date API could be deprecated soon

File: src/main/java/com/artipie/conda/Tokens.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+     * @param name User name
+     * @return Token string
+     */
+    CompletionStage<String> generate(String name);
+
+    /**
+     * Token item: username, token and expiration day.
+     * @since 0.5
+     */
+    final class TokenItem {
```
Comment (g4s8) [inline src/main/java/com/artipie/conda/Tokens.java:42]: Why this class don't have any method?

**Review by g4s8 — CHANGES_REQUESTED** [review]
@olenagerasimova please check my comments

**Comment by olenagerasimova** [comment]
@g4s8 thanks, made changes, take a look please

**Review by g4s8 — APPROVED** [review]
@olenagerasimova thanks, looks good

**Pull request merged**